samples/cxl_host_exerciser/cxl_he_cmd.h

```diff
@@ -239,29 +242,38 @@ class he_cmd : public test_command {
     return true;
   }
 
-  void he_start_test() {
-      // start test
+  bool he_set_bias_mode() {
 
-      switch (he_bias_) {
-      case HOSTMEM_BIAS:
-          he_ctl_.bias_support = 0x0;
-          break;
+    // Target memory HOST set BIAS host
+    if (he_target_ == HE_TARGET_HOST) {
+        he_ctl_.bias_support = HOSTMEM_BIAS;
+        // Target memory FPGA set BIAS host/device
+        if (he_bias_ == HOSTMEM_BIAS) {
+            he_ctl_.bias_support = HOSTMEM_BIAS;
+        } else {
+            cerr << "Invalid configuration:Target memory host and bias device"
```

Comment:
Rewrite this error: "Wrong BIAS mode for specified target memory type"

Reply:
Fixed